Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2010-03-03
Initial Package Version: 1.1
Upstream Status:         Not submitted (LFS specific)
Origin:                  Randy McMurchy
Description:             Changes references in the documentation from the
                         hard-coded /var/heimdal directory to the FHS
                         compliant /var/lib/heimdal directory.


diff -Naur heimdal-1.3.1-orig/doc/heimdal.info heimdal-1.3.1/doc/heimdal.info
--- heimdal-1.3.1-orig/doc/heimdal.info	2009-11-21 15:44:46.000000000 +0000
+++ heimdal-1.3.1/doc/heimdal.info	2010-03-03 22:48:19.000000000 +0000
@@ -420,15 +420,15 @@
 =========================
 
 The database library will look for the database in the directory
-`/var/heimdal', so you should probably create that directory.  Make
+`/var/lib/heimdal', so you should probably create that directory.  Make
 sure the directory has restrictive permissions.
 
-     # mkdir /var/heimdal
+     # mkdir /var/lib/heimdal
 
 The keys of all the principals are stored in the database.  If you
 choose to, these can be encrypted with a master key.  You do not have to
 remember this key (or password), but just to enter it once and it will
-be stored in a file (`/var/heimdal/m-key').  If you want to have a
+be stored in a file (`/var/lib/heimdal/m-key').  If you want to have a
 master key, run `kstash' to create this master key:
 
      # kstash
@@ -663,7 +663,7 @@
 `749/tcp'.
 
 Access to the administration server is controlled by an ACL file,
-(default `/var/heimdal/kadmind.acl'.) The file has the following syntax:
+(default `/var/lib/heimdal/kadmind.acl'.) The file has the following syntax:
      principal       [priv1,priv2,...]       [glob-pattern]
 
 The matching is from top to bottom for matching principals (and if
@@ -814,7 +814,7 @@
 follows:
 
      slave# ktutil get -p foo/admin hprop/`hostname`
-     slave# mkdir /var/heimdal
+     slave# mkdir /var/lib/heimdal
      slave# hpropd
 
 The master will use the principal `kadmin/hprop' to authenticate to the
@@ -870,7 +870,7 @@
 The program that runs on the master is `ipropd-master' and all clients
 run `ipropd-slave'.
 
-Create the file `/var/heimdal/slaves' on the master containing all the
+Create the file `/var/lib/heimdal/slaves' on the master containing all the
 slaves that the database should be propagated to.  Each line contains
 the full name of the principal (for example
 `iprop/hemligare.foo.se@FOO.SE').
@@ -890,7 +890,7 @@
 
 The next step is to start the `ipropd-master' process on the master
 server.  The `ipropd-master' listens on the UNIX domain socket
-`/var/heimdal/signal' to know when changes have been made to the
+`/var/lib/heimdal/signal' to know when changes have been made to the
 database so they can be propagated to the slaves.  There is also a
 safety feature of testing the version number regularly (every 30
 seconds) to see if it has been modified by some means that do not raise
@@ -1531,7 +1531,7 @@
 
 Note that the file name is space sensitive.
 
-     # cat /var/heimdal/pki-mapping
+     # cat /var/lib/heimdal/pki-mapping
      # comments starts with #
      lha@EXAMPLE.ORG:C=SE,O=Stockholm universitet,CN=Love,UID=lha
      lha@EXAMPLE.ORG:CN=Love,UID=lha
diff -Naur heimdal-1.3.1-orig/doc/setup.texi heimdal-1.3.1/doc/setup.texi
--- heimdal-1.3.1-orig/doc/setup.texi	2009-11-21 15:41:51.000000000 +0000
+++ heimdal-1.3.1/doc/setup.texi	2010-03-03 22:48:19.000000000 +0000
@@ -113,13 +113,13 @@
 Make sure the directory has restrictive permissions.
 
 @example
-# mkdir /var/heimdal
+# mkdir /var/lib/heimdal
 @end example
 
 The keys of all the principals are stored in the database.  If you
 choose to, these can be encrypted with a master key.  You do not have to
 remember this key (or password), but just to enter it once and it will
-be stored in a file (@file{/var/heimdal/m-key}).  If you want to have a
+be stored in a file (@file{/var/lib/heimdal/m-key}).  If you want to have a
 master key, run @samp{kstash} to create this master key:
 
 @example
@@ -363,7 +363,7 @@
 as @samp{749/tcp}.
 
 Access to the administration server is controlled by an ACL file,
-(default @file{/var/heimdal/kadmind.acl}.) The file has the following
+(default @file{/var/lib/heimdal/kadmind.acl}.) The file has the following
 syntax:
 @smallexample
 principal       [priv1,priv2,...]       [glob-pattern]
@@ -527,7 +527,7 @@
 
 @example
 slave# ktutil get -p foo/admin hprop/`hostname`
-slave# mkdir /var/heimdal
+slave# mkdir /var/lib/heimdal
 slave# hpropd
 @end example
 
@@ -587,7 +587,7 @@
 The program that runs on the master is @command{ipropd-master} and all
 clients run @command{ipropd-slave}.
 
-Create the file @file{/var/heimdal/slaves} on the master containing all
+Create the file @file{/var/lib/heimdal/slaves} on the master containing all
 the slaves that the database should be propagated to.  Each line contains
 the full name of the principal (for example
 @samp{iprop/hemligare.foo.se@@FOO.SE}).
@@ -613,7 +613,7 @@
 
 The next step is to start the @command{ipropd-master} process on the master
 server.  The @command{ipropd-master} listens on the UNIX domain socket
-@file{/var/heimdal/signal} to know when changes have been made to the
+@file{/var/lib/heimdal/signal} to know when changes have been made to the
 database so they can be propagated to the slaves.  There is also a
 safety feature of testing the version number regularly (every 30
 seconds) to see if it has been modified by some means that do not raise
@@ -1323,7 +1323,7 @@
 Note that the file name is space sensitive.
 
 @example
-# cat /var/heimdal/pki-mapping
+# cat /var/lib/heimdal/pki-mapping
 # comments starts with #
 lha@@EXAMPLE.ORG:C=SE,O=Stockholm universitet,CN=Love,UID=lha
 lha@@EXAMPLE.ORG:CN=Love,UID=lha
diff -Naur heimdal-1.3.1-orig/doc/vars.texi heimdal-1.3.1/doc/vars.texi
--- heimdal-1.3.1-orig/doc/vars.texi	2009-11-21 15:44:38.000000000 +0000
+++ heimdal-1.3.1/doc/vars.texi	2010-03-03 22:51:20.000000000 +0000
@@ -3,5 +3,5 @@
 @c Variables depending on installation
 @c
 
-@set dbdir /var/heimdal
+@set dbdir /var/lib/heimdal
 @set PACKAGE_VERSION 1.3.1
diff -Naur heimdal-1.3.1-orig/kadmin/kadmind.8 heimdal-1.3.1/kadmin/kadmind.8
--- heimdal-1.3.1-orig/kadmin/kadmind.8	2009-11-21 15:41:51.000000000 +0000
+++ heimdal-1.3.1/kadmin/kadmind.8	2010-03-03 22:48:19.000000000 +0000
@@ -85,7 +85,7 @@
 Principals are always allowed to change their own password and list
 their own principal.  Apart from that, doing any operation requires
 permission explicitly added in the ACL file
-.Pa /var/heimdal/kadmind.acl .
+.Pa /var/lib/heimdal/kadmind.acl .
 The format of this file is:
 .Bd -ragged
 .Va principal
@@ -155,7 +155,7 @@
 .El
 .\".Sh ENVIRONMENT
 .Sh FILES
-.Pa /var/heimdal/kadmind.acl
+.Pa /var/lib/heimdal/kadmind.acl
 .Sh EXAMPLES
 This will cause
 .Nm
diff -Naur heimdal-1.3.1-orig/kdc/kdc.8 heimdal-1.3.1/kdc/kdc.8
--- heimdal-1.3.1-orig/kdc/kdc.8	2009-11-21 15:41:51.000000000 +0000
+++ heimdal-1.3.1/kdc/kdc.8	2010-03-03 22:48:19.000000000 +0000
@@ -77,7 +77,7 @@
 .Fl -config-file= Ns Ar file
 .Xc
 Specifies the location of the config file, the default is
-.Pa /var/heimdal/kdc.conf .
+.Pa /var/lib/heimdal/kdc.conf .
 This is the only value that can't be specified in the config file.
 .It Xo
 .Fl p ,
diff -Naur heimdal-1.3.1-orig/kdc/kstash.8 heimdal-1.3.1/kdc/kstash.8
--- heimdal-1.3.1-orig/kdc/kstash.8	2009-11-21 15:41:51.000000000 +0000
+++ heimdal-1.3.1/kdc/kstash.8	2010-03-03 22:48:19.000000000 +0000
@@ -89,7 +89,7 @@
 .El
 .\".Sh ENVIRONMENT
 .Sh FILES
-.Pa /var/heimdal/m-key
+.Pa /var/lib/heimdal/m-key
 is the default keyfile if no other keyfile is specified.
 The format of a Heimdal master key is the same as a keytab, so
 .Nm ktutil
diff -Naur heimdal-1.3.1-orig/lib/kadm5/iprop.8 heimdal-1.3.1/lib/kadm5/iprop.8
--- heimdal-1.3.1-orig/lib/kadm5/iprop.8	2009-11-21 15:41:51.000000000 +0000
+++ heimdal-1.3.1/lib/kadm5/iprop.8	2010-03-03 22:48:19.000000000 +0000
@@ -93,7 +93,7 @@
 The slaves are specified by the contents of the
 .Pa slaves
 file in the KDC's database directory, e.g.\&
-.Pa /var/heimdal/slaves .
+.Pa /var/lib/heimdal/slaves .
 This has principals one per-line of the form
 .Dl iprop/ Ns Ar slave Ns @ Ns Ar REALM
 where 
@@ -131,7 +131,7 @@
 There is a keep-alive feature logged in the master's
 .Pa slave-stats
 file (e.g.\&
-.Pa /var/heimdal/slave-stats ) .
+.Pa /var/lib/heimdal/slave-stats ) .
 .Pp
 Supported options for
 .Nm ipropd-master :
diff -Naur heimdal-1.3.1-orig/lib/krb5/krb5.conf.5 heimdal-1.3.1/lib/krb5/krb5.conf.5
--- heimdal-1.3.1-orig/lib/krb5/krb5.conf.5	2009-11-21 15:41:51.000000000 +0000
+++ heimdal-1.3.1/lib/krb5/krb5.conf.5	2010-03-03 22:48:19.000000000 +0000
@@ -513,7 +513,7 @@
 		default_domain = foo.se
 	}
 [logging]
-	kdc = FILE:/var/heimdal/kdc.log
+	kdc = FILE:/var/lib/heimdal/kdc.log
 	kdc = SYSLOG:INFO
 	default = SYSLOG:INFO:USER
 .Ed
diff -Naur heimdal-1.3.1-orig/lib/krb5/krb5.conf.cat5 heimdal-1.3.1/lib/krb5/krb5.conf.cat5
--- heimdal-1.3.1-orig/lib/krb5/krb5.conf.cat5	2009-11-21 15:44:27.000000000 +0000
+++ heimdal-1.3.1/lib/krb5/krb5.conf.cat5	2010-03-03 22:48:19.000000000 +0000
@@ -529,7 +529,7 @@
                            default_domain = foo.se
                    }
            [logging]
-                   kdc = FILE:/var/heimdal/kdc.log
+                   kdc = FILE:/var/lib/heimdal/kdc.log
                    kdc = SYSLOG:INFO
                    default = SYSLOG:INFO:USER
 
